website/docs/build/build-zapp-dev-environment-direct-install.md

```diff
@@ -71,7 +71,7 @@ git clone https://github.com/pointnetwork/template.point.git ../template.point
 cd ../template.point
 npm i && npm run build
 cd ../pointnetwork
-point-deploy ../template.point --contracts
+NODE_CONFIG_ENV=devlocal MODE=zappdev point-deploy ../template.point --contracts
```

Comment:
@Koochr it should not be necessary to include these env vars here since they are already defined and set in the `point-deploy` alias defined [here](https://github.com/pointnetwork/pointnetwork/blob/develop/.bash_alias#L15).